### ISSUE TITLE
expect.toThrow() is broken for non spy methods

### ISSUE THREAD

**Issue by jurna** [djkf/jest-sinon#4]
This simple test case gives error if jest-sinon is enabled in jest:
```
describe('jest-sinon issue', () => {
  it('shoud handle exception', () => {
    expect(() => {
      throw new Error('error thrown');
    }).toThrow('error thrown');
  });
});
```
With jest-sinon enabled it gives error: TypeError: expected.threw is not a function .
Without jest-sinon everything works as expected (test passes).

**Comment by djkf** [comment]
Thanks for reporting this. I'll take a look and try get a fix out asap.

**Comment by djkf** [comment]
this should be fixed in v1.0.1. Let me know if you still have problems.
